index.html

```diff
@@ -990,32 +656,24 @@ <h2>
     Examples
   </h2>
   <pre class="example js" title="Adjusting the number of video feeds based on CPU usage">
-    const observer = new ComputePressureObserver(computePressureCallback);
+    let timerId = -1;
+    function pressureChange(entries) {
+      clearTimeout(timerId);
 
-    observer.observe("cpu", {
-      cpuUtilizationThresholds: [0.75, 0.9, 0.5],
-      cpuSpeedThresholds: [0.5],
-    });
+      const lastEntryArray = [entries.at(entries.length - 1)];
+      timerId = setTimeout(pressureChange.bind(this, lastentryArray), 30_000);
 
-    function computePressureCallback(update) {
-      // The CPU base frequency is represented as 0.5.
-      if (update.cpuSpeed >= 0.5) {
-        // Dramatically cut down compute requirements to avoid overheating.
-        limitVideoStreams(2);
-        return;
-      }
 
-      if (update.cpuUtilization >= 0.9) {
-        limitVideoStreams(2);
-      } else if (update.cpuUtilization >= 0.75) {
-        limitVideoStreams(4);
-      } else if (update.cpuUtilization >= 0.5) {
-        limitVideoStreams(8);
-      } else {
-        // The system is in great shape. Show all meeting participants.
-        showAllVideoStreams();
+      for (const entry of entries) {
+        if (entry.state == "critical") {
```

Comment:
Maybe add some prose to explain what you're doing.
And mention other states.

Reply:
Mentioned in the example explainer

Reply:
I don't see other states and how it is explained in https://github.com/WICG/compute-pressure#:~:text=if%20(entry.state%20%3D%3D%20%22critical%22)%20%7B

Reply:
Got it.